cluster/ci/monitoring/mixins/grafana_dashboards/prow.jsonnet

```diff
@@ -36,4 +36,357 @@ dashboard.new(
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="ghproxy"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='ghproxy'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="horologium"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='horologium'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="tot"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='tot'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="needs-rebase"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='needs-rebase'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="kata-jenkins-operator"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='kata-jenkins-operator'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="jenkins-dev-operator"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='jenkins-dev-operator'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="jenkins-operator"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='jenkins-operator'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="artifact-uploader"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='artifact-uploader'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="refresh"}, "pod_name", "$1", "pod", "(.*)"))',
+      legendFormat='refresh'
+    )
+  )
+  .addTarget(
+    prometheus.target(
+      'sum(pod_name:container_fs_usage_bytes:sum{namespace="ci"} * on (pod_name, namespace) label_replace(kube_pod_labels{pod!="",label_app="prow",label_component="cherrypick"}, "pod_name", "$1", "pod", "(.*)"))',
```

Comment:
Can we do this in a loop ?

Reply:
Good Q. Let me try with another board.

Reply:
Did not get the solution myself.
Asking via slack:
https://coreos.slack.com/archives/C0VMT03S5/p1559856120041000

Reply:
I didn't find a solution as well. I asked everybody :)

Reply:
@brancz  Any ideas?

Reply:
@stevekuznetsov It seems that its not possible to do that with promQL. So we have to stick with this solution for now.

Reply:
Unfortunately this is a fault in the grafonnet library. The Grafana folks are experimenting with a new API, that would allow something like this, but it's not ready for prime time yet. This is as good as we can do for now.